e2-studio-star-rx72n-firmware/libs/rx_crc/src/rx_dmaca.c

```
@@ -0,0 +1,314 @@
+    volatile struct st_dmac0 *ch = dmac_ch(channel);
+
+    /* Enable DMA transfer for this channel. */
+    ch->DMCNT.BIT.DTE = 1;
+
+    /* Issue software request (SWREQ pulse → triggers the block transfer). */
+    ch->DMREQ.BIT.SWREQ = 1;
+
+    return k_rx_ok;
+}
+
+rx_err_t rx_dmaca_wait(uint8_t channel, uint32_t timeout_cycles)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'rx_dmaca_wait' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)